Sources/Other/LayoutHelper.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  LayoutHelper.swift
+//  CollectionKit
+//
+//  Created by Luke Zhao on 2017-09-08.
+//  Copyright © 2017 lkzhao. All rights reserved.
+//
+
+import UIKit
+
+public enum JustifyContent {
+  case start, end, center, spaceBetween, spaceAround, spaceEvenly
+}
+
+public typealias AlignContent = JustifyContent
+
+public enum AlignItem {
+  case start, end, center, stretch
+}
+
+struct LayoutHelper {
+
+  static func alignItem<SizeArray: Sequence>(alignItems: AlignItem,
+                                             startingPrimaryOffset: CGFloat,
+                                             spacing: CGFloat,
+                                             sizes: SizeArray,
+                                             secondaryRange: ClosedRange<CGFloat>)
+    -> [CGRect] where SizeArray.Iterator.Element == CGSize {
+    var frames: [CGRect] = []
+    var offset = startingPrimaryOffset
+    for cellSize in sizes {
+      let cellFrame: CGRect
+      switch alignItems {
+      case .start:
+        cellFrame = CGRect(origin: CGPoint(x: offset, y: secondaryRange.lowerBound), size: cellSize)
+      case .end:
+        cellFrame = CGRect(origin: CGPoint(x: offset,
+                                           y: secondaryRange.upperBound - cellSize.height),
```

Comment:
Use a local variable for `secondaryRange.uppperBound - cellSize.height`

No need to recalculate this one for every for loop

Reply:
cellSize is only available inside the loop.